src/lib/ui/base/Header.svelte

```diff
@@ -111,62 +113,29 @@
 	</div>
 	<div
 		class="fr-header__menu fr-modal"
-		class:fr-modal--opened={$isMenuOpened}
 		id="modal-menu"
+		bind:this={modalEl}
 		aria-labelledby="fr-btn-menu-mobile"
 	>
-		<div class="fr-container">
-			<button
-				class="fr-link--close fr-link"
-				aria-controls="modal-menu"
-				on:click={() => {
-					$isMenuOpened = false;
-				}}>Fermer</button
-			>
-			{#if $isMenuOpened}
-				<div class="fr-header__menu-links">
-					{#if $session.user}
-						<ul class="fr-links-group">
-							{#if ['professional', 'particulier'].includes($session.user.role)}
-								<li>
-									<Link
-										classNames="fr-link"
-										href={`${baseUrlForRole($session.user.role)}/moncompte`}>Mon Compte</Link
-									>
-								</li>
-							{/if}
-							<li><button class="fr-link" on:click={logout}>Déconnexion</button></li>
-						</ul>
-					{/if}
-				</div>
-			{/if}
+		<div class="fr-container" on:click={closeDsfrModal}>
```

Comment:
```suggestion
<!-- hack: @lionelb: manually close modal since it's not handle by dsfr component in a spa context    -->
		<div class="fr-container" on:click={closeDsfrModal}>
```